docs/developer-guide.md

```diff
@@ -0,0 +1,55 @@
+# Developer Guide
+
+This guide documents how to start developing for Haveno.
+
+## Installing Haveno
+
+In order to develop for Haveno, first [install and run a local Haveno test network](installing.md).
+
+## Importing Haveno into Eclipse development environment
+
+The following steps describe how to set up the Eclipse development environment for Haveno. You can also develop using [IntelliJ IDEA](./idea-import.md) or VSCode if you prefer.
+
+> Note: use default values unless specified otherwise.
+
+1. If you haven't already, clone Haveno projects:
+    1. `cd ~/git/`
+    2. `git clone https://github.com/haveno-dex/haveno.git`
+    3. `git clone https://github.com/haveno-dex/haveno-ui-poc.git`
+2. Download and run the [Eclipse](https://www.eclipse.org/downloads/) installer.
+3. Select "Eclipse IDE for Enterprise Java and Web Developers" to install.
+4. Launch an eclipse workspace and close out of the welcome screen.
+5. [Download](https://search.maven.org/search?q=g:org.projectlombok%20AND%20a:lombok&core=gav) the latest version of the lombok jar.
+6. Run lombok jar, e.g.: `java -jar ~/Downloads/lombok-1.18.22.jar`.
+7. Follow prompts to install lombok to your Eclipse installation.
+8. Restart Eclipse.
+9. File > Import... > Existing Gradle Project.
+10. Select the location of the haveno project to import, e.g. ~/git/haveno.
+11. Advance to finish importing haveno project.
+12. File > Import... > Existing Projects into Workspace.
+13. Select location of haveno-ui-poc project to import, e.g. ~git/haveno-ui-poc.
+14. Advance to finish importing haveno-ui-poc project.
```

Comment:
Better move this section to a dedicated document. We already have a dedicated intelliJ doc; adding a guide for another IDE inside another document would be inconsistent. Furthermore, This is not something all devs will need, but only the users of this specific IDE.

A good solution could be to have a dedicated file for instructions for IDEs and move the content of `idea-import.md` there.

Reply:
Done.